__tests__/app.test.js

```diff
@@ -255,3 +255,77 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("201: When valid request body posted from a valid user, responds with comment object", () => {
+    const req = {
+      username: "mallionaire",
+      body: "Hello what a cool boardgame.",
+    };
+
+    //example of a comment
+    const comment = {
+      comment_id: 4,
+      body: "Hello what a cool boardgame.",
+      review_id: 2,
+      author: "mallionaire",
+      votes: 16,
+      created_at: "2017-11-22T12:36:03.389Z",
+    };
```

Comment:
What are you using this eg of a comment for? If it's just a reference to help you test, then great! But make sure to delete it before merging into main